cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,574 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`HYDRATOR-182` - Hydrator MapReduce or Spark jobs now support multiple
+  inputs. This will enable more efficient physical workflow generation due to the reduction
+  in the number of MapReduce or Spark programs required for a logical pipeline.
+
+- :cask-issue:`HYDRATOR-165` - Support multiple sources as input to a stage.
+
+- :cask-issue:`HYDRATOR-748` - Re-organizes batch pipeline settings to a top panel to
+  schedule a batch pipeline, add post-run actions and set pipeline resources and the engine
+  used.
+
+- :cask-issue:`HYDRATOR-712` - Added to Cask Hydrator a Batch Pipeline Configuration
+  Schedule.
+
+- :cask-issue:`TRACKER-108`, :cask-issue:`TRACKER-98` - Adds to Cask Tracker a tracker
+  meter widget in the UI, including search results page and details page. It displays a
+  metric that determines the 'truthfulness' of a dataset/stream being used in CDAP or Cask
+  Hydrator.
+
+- :cask-issue:`TRACKER-109`, :cask-issue:`TRACKER-100` - Adds a separate section for tags
+  in Cask Tracker. This lists all available tags in CDAP.
+
+- :cask-issue:`TRACKER-149`, :cask-issue:`TRACKER-105` - Adds a histogram for audit log in
+  Cask Tracker for easier visualization of usage of a dataset.
+
+New Features
+------------
+
+- :cask-issue:`CDAP-2963` - All HBase Tables created through CDAP will now have a key
+  ``cdap.version`` in the ``HTableDescriptor``.
+
+- :cask-issue:`CDAP-3368` - Add location for ``cdap-cli.sh`` to PATH in distributed CDAP
+  packages.
+
+- :cask-issue:`CDAP-3890` - Improved performance of the Dataset Service.
+
+- :cask-issue:`CDAP-4106` - Created pre-defined alert definitions in the CDAP Ambari
+  Service.
+
+- :cask-issue:`CDAP-4107` - Support HA CDAP installations in the CDAP Ambari Service.
+
+- :cask-issue:`CDAP-4109` - Support Kerberos-enabled clusters via the CDAP Ambari service.
+
+- :cask-issue:`CDAP-4110` - CDAP Auth Server is now supported in the CDAP Ambari Service
+  on Ambari clusters which have Kerberos enabled.
+
+- :cask-issue:`CDAP-4288` - Added an authorization extension backed by Apache Sentry to
+  enforce authorization on CDAP entities.
+
+- :cask-issue:`CDAP-4913` - Added a way to cache authorization policies so every
+  authorization enforcement request does not have to make a remote call. Caching is
+  configurable |---| it can be enabled by setting security.authorization.cache.enabled to
+  true. TTL for cache entries (``security.authorization.cache.ttl.secs``) as well as refresh
+  interval (``security.authorization.cache.refresh.interval.secs``) is also configurable.
+
+- :cask-issue:`CDAP-5740` - Provided access to ``Partitioner`` and ``Comparator`` classes
+  to the ``MapReduceTaskContext`` by implementing ``ProgramLifeCycle``.
+
+- :cask-issue:`CDAP-5770` - Allow setting of YARN container resources requirements for all
+  program types via preferences and runtime arguments.
+
+- :cask-issue:`CDAP-6052` - Fixed a bug that prevented the setting of ``local.data.dir``
```

Comment:
No. There are a bunch of these (notes in the wrong category). Fixed and fixing the others.